### PR TITLE
feat(bootstrap4-theme): add image based card component with CTA

### PR DESCRIPTION
**Pull request by maycomayco**
image
image
image

### CODE REVIEW

File: packages/bootstrap4-theme/src/scss/extends/_cards.scss [hunk 1]
```diff
@@ -744,3 +745,158 @@ Cards - Table of Contents
...
+    z-index: 20;
+
+    .content-bg {
+      h3 {
+        font-size: $uds-size-font-xxl;
+        line-height: 2.125rem;
+        padding: $uds-size-spacing-3 $uds-size-spacing-3 $uds-size-spacing-1;
+        margin: 0;
+      }
+      .hidden-details {
```
Comment (davidornelas11) [inline packages/bootstrap4-theme/src/scss/extends/_cards.scss]: Should the hidden details be scrollable? Adding overflow: scroll to hidden-details fixes this but I'm not sure if that is the desired effect. Something to bring up in standup, maybe?

**Review by davidornelas11 — CHANGES_REQUESTED** [review]

File: packages/bootstrap4-theme/src/scss/extends/_cards.scss (hunk 1 above)
Comment (maycomayco) [reply packages/bootstrap4-theme/src/scss/extends/_cards.scss]: Good point to talk about in the standup today

**Review by davidornelas11 — DISMISSED** [review]
Holding off on merging until Michael has some time to look it over as well

**Comment by maycomayco** [comment]
I closed this one because it was covered here https://github.com/ASU/asu-unity-stack/pull/1070

**Pull request closed without merging**